tests/regression/error/test-func-sliced.lus

```diff
@@ -17,21 +17,26 @@ function sincos(in: real) returns (sin: real; cos: real);
     ensure sin <= 0.0;
   );
 *)
+let
+   sin = in;
+   cos = in;
+tel
+
```

Comment:
Remove this body and set the function as imported:
function **imported** sincos(in: real) returns (sin: real; cos: real);